internal/web_api.go

```diff
@@ -84,18 +74,20 @@ func (a *API) Call(action string, reqPayload interface{}, respPayload interface{
 	return a.send(req, respPayload)
 }
 
+type FileUploadAPI struct{ *API }
```

Comment:
For me the name of this struct is misleading - I'd expect that it's used for file upload, not for file upload + everything that `API` provides.
Would it be possible to have this struct unexported and rename the factory function to something like `WithFileUpload`?

Reply:
Also, exported symbols should be documented